### ISSUE TITLE
Findbugs rule 'EI_EXPOSE_REP' is not active in Sonar.

### ISSUE THREAD

**Issue by patpatpat123** [spotbugs/sonar-findbugs#492]
## Issue Description

Small question regarding a warning observed when running Spotbugs with SonarQube please.

During my analysis, I am seeing the following two warning messages:

```
[WARNING] Findbugs rule 'EI_EXPOSE_REP' is not active in Sonar.
[WARNING] Findbugs rule 'EI_EXPOSE_REP2' is not active in Sonar.
```

Here is my Spotbugs and Sonar settings:

```
 <plugin>
                <groupId>org.sonarsource.scanner.maven</groupId>
                <artifactId>sonar-maven-plugin</artifactId>
                <version>3.9.1.2184</version>
                <dependencies>
                    <dependency>
                        <groupId>org.codehaus.woodstox</groupId>
                        <artifactId>woodstox-core-asl</artifactId>
                        <version>4.4.1</version>
                    </dependency>
                </dependencies>
            </plugin>
            <plugin>
                <groupId>com.github.spotbugs</groupId>
                <artifactId>spotbugs-maven-plugin</artifactId>
                <version>4.5.3.0</version>
                <configuration>
                    <outputDirectory>target/reports/findbugs</outputDirectory>
                </configuration>
            </plugin>
```

And the SonarQube server I am running is a 9.1.

May I ask what is the meaning of this warning, or should I say, how to "activate" those rules in Sonar please?

Made couple of research and it seems those two rules ```EI_EXPOSE_REP EI_EXPOSE_REP2``` are quite important.

Thank you

## Environment

| Component          | Version |
| ------------------ | ------- |
| SonarQube          | 9.1   |
| Sonar-FindBugs     | 4.0.4   |
| Maven              | 3.8   |
| Gradle             | ?????   |
| Java               | 11   |

**Comment by gtoison** [comment]
SpotBugs first runs an analysis with all the detectors enabled, then the plugin gets the report and creates SonarQube issues but only if the corresponding rule is enabled.
The warning message means that some instances of  `EI_EXPOSE_REP` and `EI_EXPOSE_REP2` were found but the rules `findbugs:EI_EXPOSE_REP` and `findbugs:EI_EXPOSE_REP2` are not enabled for the project's quality profile.

**Comment by patpatpat123** [comment]
Thanks a lot @gtoison for the prompt and clear answer!

The reason why I am raising this issue, is because I am not seeing any similar rules in SonarQube, neither activated nor deactivated. 

I have for sure in SonarQube the plugin enabled, as well as the profile set with all the rules from findbugs. (see screenshot) 

Yet, not seeing this rule at all. Could you please advise what am I missing please?

Thank you

<img width="1311" alt="Screen Shot 2022-03-04 at 10 21 21 AM" src="https://user-images.githubusercontent.com/43899031/156686929-9bfd1bcb-3fc7-455e-a632-d6d983f2f38b.png">


and 
<img width="712" alt="Screen Shot 2022-03-04 at 10 21 49 AM" src="https://user-images.githubusercontent.com/43899031/156686945-a5f6216d-8064-4c24-8c3e-865717dae2f5.png">

**Comment by gtoison** [comment]
The rules you are looking for are the 2nd and 3rd in your screenshot. The rule "key" is visible (top-right) once you have clicked on it:
![image](https://user-images.githubusercontent.com/86775455/156720424-591074c4-a9c8-4f6e-82e6-6371450b9208.png)

Then these rules need to be part of the quality profile used for your project(s). The plugin provides built-in quality profiles but they won't be enabled by default, so these rules won't be enabled by default. You can refer to the SonarQube documentation here: https://docs.sonarqube.org/latest/instance-administration/quality-profiles/

**Comment by patpatpat123** [comment]
The explanation is very clear, many thanks for your time and writings @gtoison
